### PR TITLE
Fix parsing for VDF files with duplicate keys

### PR DESCRIPTION
**Pull request by Perplex**
An example of a VDF with duplicate keys would be the items_game file from CSGO, it has multiple "items" keys. Here is a copy of it: https://raw.githubusercontent.com/SteamDatabase/GameTracking-CSGO/master/csgo/scripts/items/items_game.txt

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,24 +0,0 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
```
Comment (andygrunwald) [inline .gitignore]: Is there any reasoning behind this to delete this file?

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +0,0 @@
-language: go
```
Comment (andygrunwald) [inline .travis.yml]: Is there any reasoning behind this to delete this file?

File: LICENSE [hunk 3]
```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```
Comment (andygrunwald) [inline LICENSE]: Is there any reasoning behind this to delete this file?

**Comment by Perplex** [comment]
Looks like a commit that was meant for my version of the parser made it here, sorry about that. I'm going to close this PR and open a new one, since I deleted my original forked repo for this PR.

**Pull request closed without merging**